blog/2022-09-08-difference-between-i18n-and-i18next.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: difference-between-i18n-and-i18next
+
+title: "What is the difference between i18n and i18next?"
+
+authors: [jcizmar]
+
+image: "/img/blog/i18n-i18next-difference.jpg"
+
+description: "This article explains what is the difference between i18n and i18next"
+
+tags: [localization, internationalization, basics]
+---
+
+For many, this question has a pretty clear answer, but people, who are new to software development or who started to use
```

Comment:
too long phrase. e.g: For many, this question has a pretty clear answer. But there are software development newbies or developers who started to use i18next as the first choice.